### PR TITLE
feat: add home page with short links

### LINKED ISSUE #103: Implement Home page with easy connection links
image

### PR DESCRIPTION
**Pull request by pyphilia**
REVIEW THIS PR FIRST: 
https://github.com/graasp/graasp-mobile-builder/pull/111

I tried to use https://github.com/mrousavy/react-native-vision-camera but I couldn't make it work.

Instead I used expo-camera and other packages but I had to upgrade the `deploymentTarget`.
The camera is mocked in the simulator (and probably tests) since it doesn't exist in the simulator.

This PR does not include:
- managing the url to get the id. The URL is hardcoded for now.
- doesn't navigate to the item view

Screenshot 2023-11-17 at 14 33 46

Screenshot 2023-11-17 at 14 34 22

close #103

### CODE REVIEW

**Comment by pyphilia** [comment]
This PR contains the Login and the Home pages.
I'll check again the login process as mentioned by @alvrba

File: app.config.js [hunk 1]
```diff
@@ -20,9 +20,22 @@ module.exports = {
...
         ios: {
           useFrameworks: 'static',
+          deploymentTarget: '14.0',
         },
       },
     ],
+    [
+      'expo-camera',
+      {
+        cameraPermission: 'Allow $(PRODUCT_NAME) to access your camera.',
```
Comment (spaenleh) [inline app.config.js:30]:
```suggestion
        cameraPermission: 'Allow $(PRODUCT_NAME) to access the camera.',
```

File: app.config.js [hunk 2]
```diff
@@ -20,9 +20,22 @@ module.exports = {
...
+      },
+    ],
+    [
+      'expo-barcode-scanner',
+      {
+        cameraPermission: 'Allow $(PRODUCT_NAME) to access camera.',
```
Comment (spaenleh) [inline app.config.js:36]:
```suggestion
        cameraPermission: 'Allow $(PRODUCT_NAME) to access the camera.',
```

File: src/assets/adaptive-icon.png
Comment (spaenleh) [inline src/assets/adaptive-icon.png:1]: Would it be possible to use an svg version of the icon ? https://www.figma.com/file/9KHARYi1IfL9i6XZbYOpMb/All-the-logos?type=design&node-id=26-24&mode=design&t=oXtIWwpFpUt9jhcx-0

**Review by spaenleh — APPROVED** [review]
Thank you for this and the nice refactoring of the colour and removal of non-null assertions on translation strings.

File: src/assets/adaptive-icon.png
Comment (pyphilia) [reply src/assets/adaptive-icon.png:1]: not sure, I let that for later.

**Pull request merged**